src/software/backend/input/replay_logging/BUILD

```diff
@@ -0,0 +1,43 @@
+package(default_visibility = ["//visibility:public"])
```

Comment:
I believe @jonathanlew is going to delete this folder in https://github.com/UBC-Thunderbots/Software/pull/1576. 

I think if you change from `src/software/backend/input` to just `src/software/backend` that should prevent merge conflicts / be consistent with @jonathanlew's changes.

Reply:
Yup, software/backend/replay_logging would be great